app/src/main/java/com/aritra/notify/ui/screens/notes/homeScreen/NoteScreen.kt

```diff
@@ -341,15 +351,15 @@ fun NoteScreen(
 }
 
 @Composable
-fun NoList(image: Painter, contentDescription: String, message: String) {
```

Comment:
Don't change this composable function I have modified it because I want to use it in other screens also.

Reply:
Sure, @aritra-tech will amend a commit in this same PR, including `ktlint` fixes.

Reply:
Sure @dharamveer-gupta